.github/workflows/doc_pdf_export.yaml

```diff
@@ -0,0 +1,148 @@
+name: PS2 Documentation Export 
```

Comment:
Other workflows use - instead of _ in their names (yaml file, not here), might be better to stay consistent

Reply:
Renamed the file with commit id: https://github.com/COPRS/processing-sentinel-2/commit/bdf1e34fa205281ac9e99c2491efb25bbc8f1c58